awscodecommit/main.go

```diff
@@ -259,17 +265,28 @@ func (clients Clients) preparePullRequests() ([]*codecommit.PullRequest, error)
 
 //sendPush sends an event containing data about a git commit that was pushed to a branch
 func (clients Clients) sendCommitEvent(commit *codecommit.Commit) error {
+
 	log.Info("send Commit Event")
 
-	codecommitEvent := PushMessageEvent{
-		Commit:           commit,
-		CommitRepository: aws.String(repoNameEnv),
-		CommitBranch:     aws.String(repoBranchEnv),
-		EventSource:      aws.String("aws:codecommit"),
-		AwsRegion:        aws.String(awsRegionEnv),
+	event := cloudevents.Event{
+		Context: cloudevents.EventContextV03{
+			Type:            "AWS CodeCommit commit event",
+			Subject:         aws.String("AWS CodeCommit Event"),
+			Source:          *types.ParseURLRef(""),
+			ID:              *commit.CommitId,
```

Comment:
Can one commitID generate more than one event? If so, this wouldn't be a good value to use since the ID needs to be unique across all events from the same event source.